ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/EthereumWireProtocolConfiguration.java

```diff
@@ -91,9 +92,12 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return String.format(
-        "maxGetBlockHeaders=%s\tmaxGetBlockBodies=%s\tmaxGetReceipts=%s\tmaxGetReceipts=%s",
-        maxGetBlockHeaders, maxGetBlockBodies, maxGetReceipts, maxGetNodeData);
+    return MoreObjects.toStringHelper(this)
```

Comment:
Optional. Unit test.

Reply:
toString is always just for debugging purposes so the only time we should test it is when there's sensitive info that definitely shouldn't appear in a log (ie anything containing a password).